apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -207,21 +207,6 @@ router.post(
     if (req.body.__action === 'save_and_sync') {
       debug('Save and sync');
 
-      const editor = new FileModifyEditor({
-        locals: res.locals,
-        container,
-        filePath: paths.workingPath,
-        editContents: req.body.file_edit_contents,
-        origHash: req.body.file_edit_orig_hash,
-      });
-
-      if (!editor.shouldEdit()) {
-        res.redirect(req.originalUrl);
-        return;
-      }
-
-      editor.assertCanEdit();
```

Comment:
Behavior change: we'll now persist the draft edit regardless of whether or not the edit will be applied, and also regardless of whether `assertCanEdit` throws. Again, this should be fine, just raising it in case I missed something.

Reply:
Seems fine to me.